src/domains/job/main/components/JobDropDown.jsx

```
@@ -1,24 +1,15 @@
-import { useState } from 'react'
-
-const JobDropDown = ({ handleTabChange }) => {
-  const [selectedJobTabs, setSelectedJobTabs] = useState('job list') // 기본적으로 '구인' 탭으로 설정
-
+const JobDropDown = ({ selectedJobTabs, handleTabChange }) => {
```

ESLint

[error] 'selectedJobTabs' is missing in props validation

(react/prop-types)